docker-compose.yml

```diff
@@ -31,6 +31,10 @@ x-environment:
     LAGOON_ENVIRONMENT: 'local'
     LAGOON_ENVIRONMENT_TYPE: ${LAGOON_ENVIRONMENT_TYPE:-local}
     WEBROOT: web
+    GRAPHQL_USER_NAME: graphql_consumer
+    GRAPHQL_USER_PASSWORD: test
+    # @todo - what do we set this to?
+    BNF_SERVER_GRAPHQL_ENDPOINT: "https://localhost/graphql"
```

Comment:
I'm not sure what we do here - this doesnt work.
I've tested locally, and if i set it up to my own test domain (https://varnish.dpl-cms.orb.local/graphql) all is well

Reply:
> But yeah, I know that localhost should work

Ah, well no, it shouldn't. The PHP code runs in the PHP FPM container, which doesn't have a webserver. It should either be `http://nginx/graphql` (which our codes prohibits) or `https://https/graphql`, in order to actually talk with a HTTP service.

Reply:
I've found a way around it here:

https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/1853/commits/65a95a4f70c7ebbdb89c086c727ce93d12462ea1

And yeah, I know that the server domain doesnt exist yet :) This was a placeholder, more than anything else.

Reply:
Well, that won't work for me, these names isn't available in the containers. I need to have a chat with @arnested...

I can't see why the `varnish` container needs this env variable?

Reply:
I've tweaked it a bit so it actually works for me - but, as we talked about - we need to find a solution that works for the both of us. Let's leave it here for now :)